### PR TITLE
RAC-303: Add other themes and theme control in Illustration story

### PR DESCRIPTION
**Pull request by ValentinMumble**
**Description (for Contributor and Core Developer)**

Adding the other Akeneo products theme, and the illustrations

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by StevenVAIDIE — APPROVED** [review]

**Review by SamirBoulil — APPROVED** [review]

File: akeneo-design-system/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "@storybook/react": "^6.0.21",
     "@testing-library/jest-dom": "^5.11.3",
     "@testing-library/react": "^10.4.8",
+    "@testing-library/react-hooks": "^3.4.2",
```
Comment (ValentinMumble) [inline akeneo-design-system/package.json:41]: This was why we had problems on the `useShorcut` unit test 🤦

Comment (tseho) [reply akeneo-design-system/package.json:41]: http://gifbinge.tumblr.com/post/58762529735/facepalm

**Review by juliensnz — APPROVED** [review]

**Review by tseho — APPROVED** [review]

File: akeneo-design-system/src/hooks/useShortcut.unit.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+});
+
+test('It does not listen on other event', () => {
+  const callback = jest.fn();
+
+  renderHook(() => useShortcut(Key.Enter, callback));
+
+  fireEvent.keyDown(document, {key: 'Space', code: 'Space'});
+  expect(callback).not.toHaveBeenCalled();
+});
```
Comment (StevenVAIDIE) [inline akeneo-design-system/src/hooks/useShortcut.unit.ts:22]: There is no test with ref parameter in useShortcut, should we create a ticket to add it ?

**Review by StevenVAIDIE — COMMENTED** [review]
UserRoles

File: akeneo-design-system/src/hooks/useShortcut.unit.ts (hunk 2 above)
Comment (tseho) [reply akeneo-design-system/src/hooks/useShortcut.unit.ts:22]: yes, clearly, if there is a place where we need to have solid unit tests, it's in the DSM components.

**Comment by ValentinMumble** [comment]
> It seem that there is a problem for illustrations: Survey, UserRoles, UserGroups, Using, Julia, Server
> 
> Color isnot updated for: Channels, City, DeleteFamilyVariant,Delete, Draft, EntitiesRecord, ExportBuilder, Family, FamilyVariant, Groups, Locale, MediaFile, ProductCategory, Products, Proposals

@StevenVAIDIE yes, I'm not done yet ;)

File: akeneo-design-system/src/hooks/useShortcut.unit.ts (hunk 2 above)
Comment (StevenVAIDIE) [reply akeneo-design-system/src/hooks/useShortcut.unit.ts:22]: Created: https://akeneo.atlassian.net/browse/RAC-306

**Pull request merged**